proto/expression.proto

```diff
@@ -207,7 +207,6 @@ enum ScalarFuncSig {
 	/*arithmetic*/
 	PlusReal = 200;
 	PlusDecimal = 201;
-	PlusInt = 203;
```

Comment:
Hi, please reserve the original signature so that TiKV can be compatible with old TiDB.

Reply:
ok